indexForum.html

```diff
@@ -56,12 +56,11 @@
   </nav>
 </header>
     <main>
-        <section id="callToAction" class="text-center"> 
-            <form action="#">
-                <button id="postB" onclick="addingPost" type="submit" class="btn btn-info btn-lg shadow-none text-center" data-bs-toggle="button" autocomplete="off">Post a question </button>
+         
+            <form action="">
+                <button id="postB"  type="submit" class="btn btn-info btn-lg shadow-none text-center" data-bs-toggle="button" autocomplete="off">Post a question </button>

```

Comment:
You have post a question twice here. Maybe you should delete one or maybe you had something else in mind.